docs/getting-started/features/tamper-detection.en.md

```diff
@@ -9,76 +9,62 @@ To secure your Krux device, always verify firmware authenticity before installat
 
-- **Firmware Region:** Generates the image and the first two words.
+- **Firmware Region:** The area only filled with firmware code. It generates the memorable image and the first set of two words.
 
-- **User's Region:** Generates the last two words.
+- **User's Region:** The area used to stored encrypted mnemonics, settings and TC Code. It generates the last set of two words.
 
 <div style="text-align: center;">
     <img src="../../../img/flash_hash.bmp" alt="TC Flash Hash" width="200"/>
 </div>
 
-*Example: The blue symbol and words 'tail monkey' represent the firmware region, while 'wrestle over' reflects the user region.*
+*Example: The blue symbol and words 'tail monkey' represents the firmware region, while 'wrestle over' user's region.*
 
-Any change in the flash content results in a different image or words:
+Any change in the flash content results in a different image and words:
 
-- **Firmware Changes:** Alterations in the firmware region, including the bootloader, change the image and the first two words.
+- **Firmware Changes:** Alterations in the firmware region, including the bootloader, change the image and the first set of two words.
 
-- **User's Data Changes:** Modifications in the user's region, such as new settings or stored mnemonics, change the last two words.
+- **User's Data Changes:** Changes in the user's region, such as new settings or stored mnemonics, change the last set of two words.
 
-- ***TC Code* Changes:** Replacing the *TC Code* alters the image and all four words.
+- ***TC Code* Changes:** Replacing the *TC Code* alters the image and all sets of words.
 
-### Filling Empty Flash Blocks
+### Filling Empty Flash Memory Blocks
 
-Krux performs a memory sweep while simultaneously capturing a live feed from the camera. Whenever an empty block is found in the flash memory, Krux estimates the image's entropy by evaluating its color variance. Krux waits until minimum threshold is met, then uses the data from the image to fill these empty spaces with rich, random entropy.
+Use this to enhance tamper detection. Krux performs a memory sweep while capturing a live feed from the camera. Whenever an empty block is found in the flash memory, it uses the data from the image to fill these empty spaces with rich, random entropy. It estimates the image's entropy by evaluating its color variance waiting until a minimum threshold is met. 
 
 ### Ensuring Tamper Detection
 
-The TC Flash Hash function securely hashes the combination of the *TC Code*, UID, and flash content:
-
-`hash(TC Code,UID,Flash content)` -> Image + Words
-
-Hash properties ensure that without knowing the *TC Code*, UID, and flash content, an attacker cannot reproduce the TC Flash Hash results.
+The *TC Flash Hash* function securely hashes the combination of the *TC Code*, device's UID, and flash memory contents. The hash properties ensure that without knowing these three elements, an attacker will not be able to reproduce the *TC Flash Hash* results.
```

Comment:
Why remove the "function"?

Reply:
I've been thinking more from the user's point of view, many are scared of codes and don't even use the command line, so as we've already repeated the information several times I thought removing the function would be better.